app/scripts/models/application-settings.coffee

```diff
@@ -399,40 +305,36 @@ define [
 
     defaults: ->
 
-      server1 =
-        id: @guid()
-        name: FieldDB.FieldDBObject.application.brand + ' Localhost'
-        type: 'FieldDB'
-        url: 'https://localhost:3183'
-        serverCode: 'localhost'
-        corpusServerURL: null
```

Comment:
How are you doing this without persisting `corpusServerURL`? All of the FieldDB-oriented authentication logic that you've deleted in this commit was designed to allow a FieldDB web service to be specified by only providing an "auth server url". Under the new authentication logic you're implementing, will the user now need to specify both an "auth server url" and a "corpus server url" for FieldDB connections? If so, then [ a)I don't like that and] b) we need to change the server interface accordingly (if you haven't done so already...)